daffodil-runtime1/src/main/scala/org/apache/daffodil/infoset/InfosetWalker.scala

```diff
@@ -408,14 +413,23 @@ class InfosetWalker private (
     containerIndexStack.setTop(top + 1)
   }
 
+  @inline
```

Comment:
Much too big to inline this. Why inline something with Try and case matching ?

Reply:
I was trying to avoid scala creating an anonymous function for the by-name. Maybe that is just a pre-optimization though and maybe doesn't even matter. I'm not even exactly sure how scala implements by-name. I can remove this.

Reply:
I did look at the bytecode for this once, and inline did not (at that time) fold this proerly to avoid the by-name thunk. 

Supposedly newer scala (3.0) fixes this. So someday....

Reply:
Looking at the bytecode, I've confirmed that the function is not actually being inlined. And it looks like it is creating an anonymous `Function0`. I'll remove as this clearly doesn't do anything.